packages/fx-core/resource/package.nls.json

```diff
@@ -781,6 +781,15 @@
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
   "core.NewOfficeAddinOptionVSC.label": "Start with an Outlook add-in",
   "core.NewOfficeAddinOptionVSC.detail": "Use the Teams Toolkit to create a new Office add-in application.",
+  "core.summary.actionDescription": "Action %s%s",
+  "core.summary.lifecycleDescription": "Running lifecycle stage: %s(%s step(s) in total) The following actions will be executed %s",
+  "core.summary.lifecycleNotExecuted": "%s Lifecycle stage %s not executed.",
+  "core.summary.lifecycleSucceeded": "%s Lifecycle stage %s executed successfully.",
```

Comment:
"executed" -> "was executed"

similar as comment above, consider make 'L' lower case if the %s before it always has value.

Reply:
The %s before it will be either √, × or ! for success, failure and not executed.